pkg/didcomm/protocol/route/service.go

```diff
@@ -100,10 +100,18 @@ func (s *Service) HandleInbound(msg *service.DIDCommMsg) (string, error) {
 			if err := s.handleRequest(msg); err != nil {
 				logger.Errorf("handle route request error : %s", err)
 			}
+		case GrantMsgType:
```

Comment:
Should error message be more specific -- based on case?

Reply:
agree. missed to update this.

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/67425bcda88b47c412b8e368dd2bd5acc99555ef..2dff0266368cd86fb89d1fdd3acb54eeb0f620f8).